### PR TITLE
New theme

### PR DESCRIPTION
**Pull request by simon-dev-ml**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️❌
| New feature?  | ✔️❌
| Is backward-compatible?    | ✔️❌
| Tests pass?   | ✔️❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any
| Updated README/docs?   | ✔️❌
| Added CHANGELOG entry?   | ✔️❌

### CODE REVIEW

File: src/themes/silveredit.js [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  object_indent: true, /* Indent nested object elements */
+  object_border: false, /* Add border around object elements */
+  table_border: false, /* Add border to array "table" row and cells */
+  table_hdiv: false, /* Add bottom-border to array "table" cells */
+  table_zebrastyle: false, /* Add "zebra style" to array "table" rows */
+  input_size: 'small', /* Size of input and select elements. "small", "normal", "large" */
+  enable_compact: false
+}
+
+export class silvereditTheme extends AbstractTheme {
```
Comment (kirillrdy) [inline src/themes/silveredit.js]: do we need to overwrite all functions ? did we need to modify just the radio buttons ?

File: src/themes/silveredit.css [hunk 2]
```diff
@@ -0,0 +1,258 @@
+.slider {
```
Comment (kirillrdy) [inline src/themes/silveredit.css]: do we need to clone entire css from a theme ?

looks like its a copy of tailwind, could just import that

File: src/themes/index.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ import { barebonesTheme } from './barebones.js'
 // import  { materializeTheme } from  './materialize'
 import { spectreTheme } from './spectre.js'
 import { tailwindTheme } from './tailwind.js'
+import { silvereditTheme } from './silveredit.js'
```
Comment (kirillrdy) [inline src/themes/index.js:12]: we can add a theme without having our fork for json-editor

**Comment by kirillrdy** [comment]
main 2 points
1) there seems to be a lot of unnecessary code ( both js and css)
2) do we want to maintain our fork ? ( pros and cons )

File: src/themes/silveredit.css (hunk 2 above)
Comment (simon-dev-ml) [reply src/themes/silveredit.css]: Yeh better idea

File: src/themes/silveredit.js (hunk 1 above)
Comment (simon-dev-ml) [reply src/themes/silveredit.js]: Yeh, just radio buttons and checkboxes

File: src/themes/index.js (hunk 3 above)
Comment (simon-dev-ml) [reply src/themes/index.js:12]: How? By just maintaining  a local version of json-editor somewhere and putting build on s3?

**Comment by simon-dev-ml** [comment]
@kirillrdy Ok I've made those changes

**Comment by kirillrdy** [comment]
@simon-dev-ml  looks good, much less code !

I haven't used github in a long time,  so i guess :+1:   ?

**Pull request merged**